tdrs-frontend/.env

```diff
@@ -37,7 +37,7 @@ REACT_APP_DEBOUNCE_TIME=30000
 REACT_APP_EVENT_THROTTLE_TIME=60000
 
 # Enable the Kibana tab for dev purposes.
-# REACT_APP_DEV_KIBANA=true
+REACT_APP_DEV_KIBANA=true
```

Comment:
@elipe17 will this be reverted prior to merge?

Reply:
@ADPennington, yes. I will revert that now and re-request your approval